File: JavaScript/8js 2/sakuraFall.js
/*
|--------------------------------------------------------------------------
| The falling speed of petals
|--------------------------------------------------------------------------
|
*/

function sakuraFall(v) {
   return v > 0 ? 400 / v : 0;
}


const sakuraFall = (x) => (x <= 0 ? 0 : 400 / x);


function sakuraFall(velocity) {
   if (velocity < 1) {
      return 0;
   } else {
      let distance = 400;
      return distance / velocity;
   }
}


console.log(sakuraFall(5), 80);
console.log(sakuraFall(10), 40);
console.log(sakuraFall(-1), 0);

File: Python/8py 2/sakura_fall.py
"""
|--------------------------------------------------------------------------
| The falling speed of petals
|--------------------------------------------------------------------------
|
"""

def sakura_fall(v):
   return 400 / v if v > 0 else 0


def sakura_fall(velocity):
   if velocity < 1:
      return 0
   else:
      distance = 400
      return distance / velocity


def sakura_fall(v):
   return 0 if v <= 0 else 400 / v


print(sakura_fall(5), 80)
print(sakura_fall(10), 40)
print(sakura_fall(-1), 0)